Source/Core/Core/HW/DSP.cpp

```diff
@@ -603,6 +603,13 @@ void WriteARAM(u8 value, u32 address)
   s_ARAM.ptr[address & s_ARAM.mask] = value;
 }
 
+bool HostIsARAMAddress(u32 address)
+{
+  if (s_ARAM.wii_mode && (address & 0x10000000) == 0)
+    return Memory::GetPointer(address & Memory::RAM_MASK) != nullptr;
```

Comment:
GetPointer shows a panic alert for invalid addresses, so do we really want to use it here?

Reply:
I replaced it with the proper check instead.